tests/sentry/tasks/integrations/test_link_all_repos.py

```diff
@@ -0,0 +1,204 @@
+from datetime import timedelta
+from unittest.mock import patch
+
+import pytest
+import responses
+from django.db import IntegrityError
+from django.utils import timezone
+
+from sentry.integrations.github.integration import GitHubIntegrationProvider
+from sentry.models.repository import Repository
+from sentry.shared_integrations.exceptions.base import ApiError
+from sentry.snuba.sessions_v2 import isoformat_z
+from sentry.tasks.integrations.link_all_repos import link_all_repos
+from sentry.testutils.cases import IntegrationTestCase
+from sentry.testutils.silo import region_silo_test
+
+
+@region_silo_test(stable=True)
+class LinkAllReposTestCase(IntegrationTestCase):
+    provider = GitHubIntegrationProvider
+    base_url = "https://api.github.com"
+    key = "github"
+
+    def setUp(self):
+        super().setUp()
+        self.installation_id = "github:1"
+        self.user_id = "user_1"
+        self.app_id = "app_1"
+        self.access_token = "xxxxx-xxxxxxxxx-xxxxxxxxxx-xxxxxxxxxxxx"
+        self.expires_at = isoformat_z(timezone.now() + timedelta(days=365))
+
+    def _add_responses(self):
+        responses.add(
+            responses.POST,
+            self.base_url + f"/app/installations/{self.installation_id}/access_tokens",
+            json={"token": self.access_token, "expires_at": self.expires_at},
+        )
+        responses.add(
+            responses.GET,
+            self.base_url + "/installation/repositories?per_page=100",
+            status=200,
+            json={
+                "total_count": 2,
+                "repositories": [
+                    {
+                        "id": 1,
+                        "full_name": "getsentry/sentry",
+                    },
+                    {
+                        "id": 2,
+                        "full_name": "getsentry/snuba",
+                    },
+                ],
+            },
+        )
+
+    @patch("sentry.integrations.github.client.get_jwt", return_value=b"jwt_token_1")
```

Comment:
All but 1 test case has this mock. You can wrap the class instead and ignore it for the one test you don't need it.